ansible/roles/docker/templates/ubuntu1804.Dockerfile.j2

```diff
@@ -24,7 +24,8 @@ RUN apt-get update && apt-get dist-upgrade -y && apt-get install -y \
     curl \
     python-pip \
     python3-pip \
-    libfontconfig1
+    libfontconfig1 \
+    libcap-dev
```

Comment:
Since the cross compile job is still failing to find `sys/capability.h` I presume the headers are being picked up from the cross compiler (https://github.com/rvagg/rpi-newer-crosstools) rather than the container.

Reply:
Sorry, you've lost me here :) Could you explain what, if anything, we can do get this to work?

Reply:
Sorry, I have no idea how to get it to work 😞. I suspect for cross compiling the system headers can't be used because they will be for the host architecture.

Reply:
I have an idea which is to copy the header to the sysroot that the cross compiler is using. I've tried this out locally using a container and it can then compile without complaining about not finding the header. This is an update to the DockerFile and I'm wondering how I could try this out. I'm guessing that just re-running the CI job would not be enough?

Reply:
Yeah, we'd need to rebuild the Docker image and restart the container.
@rvagg @sxa Any concerns over modifying the sysroot for the ARM cross-compiler?

Reply:
uhh, yeah that looks a bit weird, is libcap implemented simply as a header or is there a library behind it too? I think by copying that header into there says that you have a arm-rpi-linux-gnueabihf backing for whatever's in this header, is that true? has this been tested? These are going to be used for armv7l binary release builds, are we requiring users to have libcap installed to use our binaries everywhere now?

Reply:
> uhh, yeah that looks a bit weird, is libcap implemented simply as a header or is there a library behind it too?

Yeah, sorry about that. I was not sure how to get the header into the sysroot and this was the only thing I could think of.

We only need the header in this case and we are not linking with libcap, so the requirement for someone building Node.js would be that they have the libcap-dev package installed.

Reply:
I guess if it's not being linked then that's a positive, but what about the contents of capability.h? I've had a quick look and I don't see anything obvious that might result in differences between architectures—is there any risk that we're going to be linking types or sizes that are going to be wrong for the arch we're cross compiling? I guess that's the most important question to answer and maybe it requires some testing of cross compiled binaries hitting the codepaths that are using this header?

Reply:
I've taken a look at just using a `syscall` instead of using the function definition in `sys/capabilities.h` and perhaps that would avoid this trouble. I'm running CI tests for this now (https://ci.nodejs.org/job/node-test-pull-request/39333/).